GrowthBook/src/commonMain/kotlin/com/sdk/growthbook/GrowthBookSDK.kt

```diff
@@ -23,8 +23,7 @@ import com.sdk.growthbook.utils.toHashMap
 import kotlinx.coroutines.flow.Flow
 import kotlinx.serialization.json.JsonObject
 
-typealias GBTrackingCallback = (GBExperiment, GBExperimentResult) -> Unit
-typealias GBExperimentRunCallback = (GBExperiment, GBExperimentResult) -> Unit
+typealias GBTrackingCallback = (GBExperiment, GBExperimentResult?) -> Unit
```

Comment:
Hello! We noticed that `GBExperimentResult` became nullable. 
In which cases can null arrive and why? Because also is not mentioned in changelog for v1.1.63

Reply:
In order to answer the question: null can arrive in result of remote evaluation.
It is good that you have noticed this change. It seems like in JS SDK the second parameter is not nullable.
The pull request to rollback was created:
https://github.com/growthbook/growthbook-kotlin/pull/154/files